common/views/components/PageHeader/PageHeader.js

```diff
@@ -154,7 +154,8 @@ const PageHeader = ({
 
       {HeroPicture &&
         <div className={classNames({
-          'relative': true
+          'relative': true,
+          [spacing({s: 10}, {padding: ['bottom']})]: true
```

Comment:
Why's this inherited some spacing rather than using the components?

Reply:
The `* + *` selector needs the things it operates on to be adjacent in the DOM. I guess we could refactor the hero image into the body to accommodate this?

I did it this way for the sake of my sanity (but will add a `TODO`).